orc8r/cloud/go/services/eventd/obsidian/handlers/handlers.go

```diff
@@ -53,8 +56,45 @@ const (
 	elasticFilterHardwareID = "hw_id" + dotKeyword
 	elasticFilterEventTag   = "event_tag" + dotKeyword // We use event_tag as fluentd uses the "tag" field
 	elasticFilterTimestamp  = "@timestamp"
+
+	ManageNetworkPath  = obsidian.V1Root + "networks" + obsidian.UrlSep + ":network_id"
+	LogSearchQueryPath = ManageNetworkPath + obsidian.UrlSep + "logs" + obsidian.UrlSep + "search"
+	LogCountQueryPath  = ManageNetworkPath + obsidian.UrlSep + "logs" + obsidian.UrlSep + "count"
 )
 
+// GetObsidianHandlers returns all the obsidian handlers for eventd.
+func GetObsidianHandlers() []obsidian.Handler {
+	var ret []obsidian.Handler
+
+	// Elastic
+	elasticConfig, err := config.GetServiceConfig(orc8r.ModuleName, "elastic")
+	if err != nil {
+		ret = append(ret, obsidian.Handler{Path: LogSearchQueryPath, Methods: obsidian.GET, HandlerFunc: getInitErrorHandler(err)})
```

Comment:
Also we should probably be setting error handlers for all the paths, I understand this is old code, but I guess this might not have been updated.